File: src/example1.c
#include "example1.h"
double cubic_poly(double x,
                  double c0,
                  double c1,
                  double c2,
                  double c3) {
  return c0 + c1 * x + c2 * x*x + c3 * x*x*x;
}

File: src/example1.py
#!/usr/bin/env python3.10

# Setup search path:
import sys ; sys.path.append('target/python')

# Load SWIG bindings:
import example1_swig as example1

# Use SWIG bindings:
print("EXAMPLE1_VERSION = " + example1.EXAMPLE1_VERSION)
print(example1.cubic_poly(2.0, 3.0, 5.0, 7.0, 11.0))
